<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ranjeet-MacBook-Pro.local" failures="1" tests="4" name="Test" time="79.262" errors="0" timestamp="2025-01-27T21:49:19 IST">
  <testcase classname="testCases.TC02_LoginTest" name="verify_Login" time="17.746">
    <skipped/>
  </testcase> <!-- verify_Login -->
  <testcase classname="testCases.TC02_LoginTest" name="verify_Login" time="16.786">
    <skipped/>
  </testcase> <!-- verify_Login -->
  <testcase classname="testCases.TC02_LoginTest" name="verify_Login" time="16.078">
    <skipped/>
  </testcase> <!-- verify_Login -->
  <testcase classname="testCases.TC02_LoginTest" name="verify_Login" time="16.172">
    <failure type="java.lang.AssertionError" message="Failed to navigate to &amp;apos;My Account&amp;apos; page. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to navigate to 'My Account' page. expected [true] but found [false]
at testCases.TC02_LoginTest.verify_Login(TC02_LoginTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verify_Login -->
</testsuite> <!-- Test -->
